tests/test_models.py

```diff
@@ -1135,6 +1135,9 @@ def test_load_config_in_charmcraft_yaml(tmp_path, prepare_charmcraft_yaml):
                 test-bool:
                   default: true
                   type: boolean
+                test-secret:
+                  default: secret:co1s9mnmp25c762drvtg
```

Comment:
So the secret string begins with `secret:`?

Reply:
Yes, e.g.:

```
$ juju add-secret example foo=bar
secret:co6tjmvmp25c77dhf980
```

```
$ juju config secretconfig secretopt
secret:co6tjmvmp25c77dhf980
```

```
$ juju config secretconfig secretopt=co6tjmvmp25c77dhf980
{}
ERROR updating charm config settings: option "secretopt" expected secret, got "co6tjmvmp25c77dhf980"
```